gemma-web/src/main/java/ubic/gemma/web/controller/expression/experiment/ExperimentalDesignControllerImpl.java

```diff
@@ -656,6 +656,10 @@ public void updateExperimentalFactors( ExperimentalFactorValueObject[] efvos ) {
     @Override
     public void updateFactorValueCharacteristics( FactorValueValueObject[] fvvos ) {
 
+        /*
```

Comment:
The simplest option would be to add 4 fields in the VO and mark them with `@GemmaWebOnly`. At some future point, the REST API should have it's own VOs.

Reply:
Ok, it's done, I've added the fields. I'll finish the UI tomorrow!